### PR TITLE
User views

### PR DESCRIPTION
**Pull request by Ti-amo**
- User Profile

image

- Edit Profile

image

- Change Password
image

### CODE REVIEW

**Review by zovivo — APPROVED** [review]

**Review by zovivo — COMMENTED** [review]
2 commit kìa 😁

**Review by nguyenvietngocquang — APPROVED** [review]

**Comment by Ti-amo** [comment]
ready

File: src/main/webapp/views/users/edit.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+			<!-- Breadcrumb -->
+			<!-- <nav aria-label="breadcrumb" class="main-breadcrumb">
+					<ol class="breadcrumb">
+						<li class="breadcrumb-item"><a href="index.html">Home</a></li>
+						<li class="breadcrumb-item"><a href="javascript:void(0)">User</a></li>
+						<li class="breadcrumb-item active" aria-current="page">User
+							Profile</li>
+					</ol>
+				</nav> -->
+			<!-- /Breadcrumb -->
```
Comment (vanvtt-0952) [inline src/main/webapp/views/users/edit.html]: Xóa mạnh tay chứ comment chi nữa bé

**Comment by Ti-amo** [comment]
ready

**Pull request merged**